bin/package.boilerplate.json

```diff
@@ -1,9 +1,10 @@
 {
   "devDependencies": {
-    "poetic": "^1.0.5"
+    "poetic": "^1.0.5",
+    "yarpm": "^0.2.1"
   },
   "scripts": {
-    "code:clean": "yarn code:lint && yarn code:format",
+    "code:clean": "yarpm code:lint && yarpm code:format",
```

Comment:
This can be set at installation time to `yarn <>` or `npm run <>`. The extra package is not necessary.  Feel free to update the installation script.

Reply:
I changed this in the newest commit to use `${manager}` as a place holder until it finds out which dependency manager the user has installed.